### PR TITLE
Add packages of bwein-net

### PR DESCRIPTION
**Pull request by rabauss**
add [bwein-net/contao-database-backup](https://packagist.org/packages/bwein-net/contao-database-backup) + [bwein-net/contao-protected-redirect-element](https://packagist.org/packages/bwein-net/contao-protected-redirect-element)

### CODE REVIEW

**Comment by Toflar** [comment]
I'll check the issues and fix them myself.

File: meta/bwein-net/contao-database-backup/de.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+de:
+    title: Datenbank-Backup
+    description: >
+        Diese Erweiterung bietet ein Konsolen-Kommando um einfach Backups von der Contao-Datenbank zu erstellen.
```
Comment (xchs) [inline meta/bwein-net/contao-database-backup/de.yml]:
```suggestion
        Diese Erweiterung bietet ein Konsolen-Kommando, um einfach Backups von der Contao-Datenbank zu erstellen.
```

**Comment by xchs** [comment]
@rabauss Could you also update the [CODEOWNERS](https://github.com/contao/package-metadata/blob/master/CODEOWNERS) file?

**Comment by Toflar** [comment]
Okay, master is clean again, sorry guys and happy new year!

**Comment by xchs** [comment]
Thanks a lot! Happy New Year, too.

**Pull request merged**